packages/fuels/tests/logs.rs

```diff
@@ -439,5 +441,187 @@ async fn test_multi_call_require_log_multi_contract() -> Result<(), Error> {
 
         assert_is_revert_containing_msg("StructDeeplyNestedGeneric", error);
     }
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_script_get_logs() -> Result<(), Error> {
+    // ANCHOR: script_logs
+    script_abigen!(
+        log_script,
+        "packages/fuels/tests/logs/script_logs/out/debug/script_logs-abi.json"
+    );
+
+    let wallet = launch_provider_and_get_wallet().await;
+    let bin_path = "../fuels/tests/logs/script_logs/out/debug/script_logs.bin";
+    let instance = log_script::new(wallet.clone(), bin_path);
+
+    let response = instance.main().call().await?;
+
+    let logs = response.get_logs()?;
+    let _log_u64 = response.get_logs_with_type::<u64>()?;
```

Comment:
```suggestion
    let log_u64 = response.get_logs_with_type::<u64>()?;
```

Reply:
I am not using this field in this test.. but I left it there for the book example. This way I can show both and do not have to make a new anchor.. If we do not like this I will add the second anchor ...